feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -122,12 +123,30 @@ class HomeViewModel @Inject constructor(
         }
     }
 
-    private fun showWillFeedDialog() {
-        updateState { copy(willFeedState = WillFeedState(true)) }
+    private fun hideAllFeedingPointsButOne(selectedFeedingPoint: FeedingPointModel) {
+        updateState {
+            copy(feedingPoints = persistentListOf(selectedFeedingPoint))
+        }
     }
 
-    private fun dismissWillFeedDialog() {
-        updateState { copy(willFeedState = WillFeedState(false)) }
+    private fun saveUserAsCurrentFeeder(feedingPointId: Int) {
+        viewModelScope.launch {
+            saveUserAsFeederUseCase(feedingPointId).collect { isSuccessful ->
+                if (isSuccessful) {
+                    updateState {
+                        copy(feedingPoints = state.feedingPoints)
```

Comment:
What is the purpose of this action?

Reply:
Nothing I guess, Should I print logs indicating if the result is successful or not?

Reply:
I would rather do all actions sequentially. Start route only if saving is successful